Host/Controllers/ImageController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using Domain.Persistance;
+using Host.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Host.Controllers
+{
+    public class ImageController : ControllerBase
```

Comment:
Add route directly for controller
```suggestion
[Route(RouteConstants.Base)]
public class ImageController : ControllerBase
```